src/pages/Mentor/Messages/index.js

```diff
@@ -23,45 +24,47 @@ export default ({
   }, [messages && messages.length]);
 
   return (
-    <div className="messages" onScroll={onScroll} ref={messagesRef}>
-      <div className="messages-content">
-        {messages &&
-          messages
-            .filter((message) => message.type === "m.room.message")
-            .map((message, index) => {
-              let shouldDisplayDayMarker = false;
-              if (messages[index - 1]) {
-                shouldDisplayDayMarker = getAreDatesOnDifferentDays(
-                  messages[index - 1].origin_server_ts,
-                  message.origin_server_ts
+    <>
+      <Container className="messages">
+        <div className="messages-content" onScroll={onScroll} ref={messagesRef}>
+          {messages &&
+            messages
```

Comment:
Make habbit of using ?. syntax here for array destructure  or the filter thing its more of cleaner approch

Reply:
It was not done by me. But we'll fix this.